src/test/java/firstSimpleJavaTasks/MaxMinAverageTests.java

```diff
@@ -0,0 +1,48 @@
+package firstSimpleJavaTasks;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class MaxMinAverageTests {
+
+    private int[] newArray = {0,1,2,3,4,5,6,7,8,9};
+    private int[] zeroLengthArray = {};
+    private int[] arrayWithNegativeValues = {-5,225,-1,0,25,-61};
```

Comment:
Try to keep consistency in naming, `arrayWithNegativeValues ` and  `newArray` - put `array` into different places.
It would be better to use a similar approach.

Reply:
Array  variables renamed using single naming approach